lib/generator/utils.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Thrift.Generator.Utils do
+  def merge_blocks([{:__block__, _, contents} | rest]) do
+    merge_blocks(contents) ++ merge_blocks(rest)
+  end
+
```

Comment:
I've been following this style suggestion re: grouping function definitions:

https://github.com/rrrene/elixir-style-guide#group-function-definitions